server/src/internalClusterTest/java/org/elasticsearch/aliases/IndexAliasesIT.java

```diff
@@ -324,27 +316,15 @@ public void testSearchingFilteringAliasesSingleIndex() throws Exception {
         );
 
         logger.info("--> checking single non-filtering alias search");
-        assertResponse(
-            prepareSearch("alias1").setQuery(QueryBuilders.matchAllQuery()),
-            searchResponse -> assertHits(searchResponse.getHits(), "1", "2", "3", "4")
-        );
-
         logger.info("--> checking non-filtering alias and filtering alias search");
-        assertResponse(
-            prepareSearch("alias1", "foos").setQuery(QueryBuilders.matchAllQuery()),
-            searchResponse -> assertHits(searchResponse.getHits(), "1", "2", "3", "4")
-        );
-
         logger.info("--> checking index and filtering alias search");
-        assertResponse(
-            prepareSearch("test", "foos").setQuery(QueryBuilders.matchAllQuery()),
-            searchResponse -> assertHits(searchResponse.getHits(), "1", "2", "3", "4")
-        );
-
         logger.info("--> checking index and alias wildcard search");
```

Comment:
I think it's fine to delete these log lines, there's little to no value in them now that they don't correspond to exactly when the queries run anymore.

Reply:
Thank you @original-brownbear. I will proceed with those deletions before merging. I am on the same page